nservicebus/sqlserver/multiple-databases.md

```diff
@@ -1,21 +1,66 @@
+In the default SQL Server transport setup, the messages and other data are stored in multiple tables in a single database. The default schema is `dbo`, but it is possible to [specify a custom schema](/nservicebus/sqlserver/configuration.md#custom-database-schemas).
 
- * different schemas in a single database
- * different databases in a single SQL Server instance
- * different SQL Server instances
+In some environments it might be necessary to use a more advanced setup. The supported options are:
+ * ***default***: single schemas in a single database (Versions 1.x and higher)
+ * ***multi-schema***: multiple schemas in a single database (Versions 2.1 and higher)
+ * ***multi-catalog***: multiple databases in a single SQL Server instance (Versions 2.1 and higher)
+ * ***multi-instance***: multiple databases in various SQL Server instances (Versions 2.1 to 2.x).
 
-The transport will route messages to destination endpoints based on the configuration. If no specific configuration has been provided for a particular destination endpoint, the transport assumes the destination has the same configuration (schema, database and instance name/address) as the sending endpoint. If this assumption turns out to be false (the transport cannot connect to destination queue), an exception is thrown immediately. There is no store-and-forward mechanism on the transport level (and hence -- no dead-letter queue).
+The transport will route messages to destination endpoints based on the configuration. If no specific configuration has been provided for a particular destination endpoint, the transport assumes the destination has the same configuration (schema, database and instance name) as the sending endpoint. If the destination endpoint has a different configuration and it hasn't been provided, then exception will be thrown immediately (since the transport cannot connect to the destination queue).
 
-NOTE: If the destination endpoint uses a different database or server instance, sending a message to it might cause the transaction to escalate to a distributed transaction which may not be desirable. Using the [Outbox](/nservicebus/outbox/ feature, DTC escalations can be avoided as long as the both the endpoint's Outbox and business data share the same database.
+WARNING: SQL Server transport does not support store-and-forward mechanism on a transport level, so there is no dead-letter queue. It means that in *multi-catalog* and *multi-database* modes if a remote endpoint's infrastructure (e.g. DTC or SQL Server instance) is down, then messages can't be delivered to it. That makes this particular endpoint, and all endpoints depending on it, unavailable. That problem can be addressed by using [Outbox](/nservicebus/outbox/) feature, as explained in [*Multi-catalog* and *multi-instance* with store-and-forward](#multi-catalog-and-multi-instance-with-store-and-forward) section. 
```

Comment:
Do we need to mention DLQ here. For me it's a bit misleading but it might be only me.